jwst/extract_1d/ifu.py

```diff
@@ -1064,3 +1076,26 @@ def shift_ref_image(mask, delta_y, delta_x, fill=0):
     temp[..., oslice_y, oslice_x] = mask[..., islice_y, islice_x]
 
     return temp
+
+
+def copy_keyword_info_ifu( source_type, x_center, y_center, slitname , spec: SpecModel):
+    """Copy metadata from the input IFU cube to the output spectrum.
+
+    Parameters
+    ----------
+    input_model : An IFUCubeM object
+        Metadata will be copied from the input IFUcube to output `spec`.
+
+    slitname : str or None
+        The name of the slit.
+
+    spec : One element of MultiSpecModel.spec
+        Metadata attributes will be updated in-place.
+
+    """
+    if slitname is not None and slitname != "ANY":
```

Comment:
I think this duplicates what's already been done up above around line 156 right before this function is called.